homeassistant/components/zwave_js/services.yaml

```diff
@@ -29,6 +29,56 @@ set_lock_usercode:
       selector:
         text:
 
+set_lock_configuration:
+  target:
+    entity:
+      domain: lock
+      integration: zwave_js
+  fields:
+    operation_type:
+      required: true
+      example: timed
+      selector:
+        select:
+          options:
+            - constant
+            - timed
+    lock_timeout:
+      required: false
+      example: 1
+      selector:
+        text:
```

Comment:
Why is this a text selector when it's a number?

Reply:
Because there's technically no max, at least as far as I am aware (@kpine or @AlCalzone is this accurate?) but a number selector expects one

Reply:
Nvm it's a 16 bit unsigned value so I guess `65535` is technically the max